### PR TITLE
cmake: add FindALSA.cmake

### PR DESCRIPTION
**Pull request by jobo-zt**

### CODE REVIEW

**Comment by jobo-zt** [comment]
fix code style.

**Comment by sreimers** [comment]
FindALSA is provided by cmake:

https://cmake.org/cmake/help/v3.10/module/FindALSA.html

Why is a custom one needed?

File: include/baresip.h [hunk 1]
```diff
@@ -1699,7 +1699,7 @@ int  peerconnection_new(struct peer_connection **pcp,
 		        const struct rtc_configuration *config,
 		        const struct mnat *mnat, const struct menc *menc,
 		        peerconnection_gather_h *gatherh,
-		        peerconnection_estab_h,
+		        peerconnection_estab_h *estabh,
```
Comment (sreimers) [inline include/baresip.h:1702]: Unrelated change

**Comment by jobo-zt** [comment]
Okay, close

**Comment by jobo-zt** [comment]
I compiled the alsa module in my raspberry, but it was not found by default, so I wrote it. I think I can close it.
